src/app/modules/domain/knowledge-map/knowledge-map.component.html

```diff
@@ -2,11 +2,11 @@
   <button mat-menu-item [routerLink]="['/kc',unitId,kc.id]" [matTooltip]="kc.name" [matTooltipDisabled]="expanded" [matTooltipPosition]="'after'"
       [disabled]="isLocked(kc)"
       [ngClass]="{'kc-in-progress': isInProgress(kc)}">
-    <span [ngStyle]="{'padding-left': 16*level+'px'}" class="kc-menu">
+    <span [ngStyle]="{'padding-left': 33*level+'px'}" class="kc-menu">
       <mat-icon *ngIf="isLocked(kc)">lock</mat-icon>
       <mat-icon *ngIf="isCompleted(kc)">done</mat-icon>
       <span *ngIf="isInProgress(kc)">
-        <b class="mastery">{{kc.mastery.mastery * 100}}</b>
+        <b class="mastery">{{(kc.mastery.mastery * 100).toFixed(2)}}</b>
```

Comment:
If we are storing mastery scores at two decimals, this means that we will have on the backend e.g., 0.85. This will always be a full % number on the front-end, so no need for toFixed.